install_common_crypto.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/sh
 
+COMMON_CRYPTO_DIR="${SDKROOT}/usr/include/CommonCrypto"
+if [ -f "${COMMON_CRYPTO_DIR}/module.modulemap" ]; then
+    exit 0
```

Comment:
for consistency, maybe we should add a log here, or remove the other one below 💄

Reply:
Fixed.